src/pages/components/typography.mdx

```diff
@@ -16,6 +17,17 @@ components:
 
 ## How to use it
 
+### Block quotation
```

Comment:
Is this the right terminology? I've typically heard these referred to simply as `block quote`.

Reply:
@allisonacs please see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote where we learn this is the "Block Quotation" element. But I'm fine to update, if necessary.

Reply:
huh, TIL. 

I understand wanting to mirror the standard, but it feels like esoteric language to me. My personal vote is to use a more colloquial term for it, but I get that I am kind of picking/choosing where I want to apply that standard. Up to you 🙂

Reply:
colloquial is king 👑